### PR TITLE
Feature/hw 3

### PR DESCRIPTION
**Pull request by LevapVeeskela**
catch)

### CODE REVIEW

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_03/BaseTaskService.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        protected UserValuesModel UserValues;
+
+        public BaseTaskService()
+        {
+            UserValues = new UserValuesModel
+            {
+                FirstValue = GetValueFromConsole(HomeWorkThree.TextInfoFirstValueInfo),
+                SecondValue = GetValueFromConsole(HomeWorkThree.TextInfoSecondValueInfo)
+            };
+        }
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/BusinessLogic/Services/HW_03/BaseTaskService.cs]: one  empty line there

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_03/BaseTaskService.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Common.Constants;
+using Common.Models;
+using static System.String;
+
+namespace BusinessLogic.Services.HW_03
+{
+    public class BaseTaskService
+    {
+        protected UserValuesModel UserValues;
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/BusinessLogic/Services/HW_03/BaseTaskService.cs]: I would replace to property instead of field declaration

Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_03/BaseTaskService.cs]: ok

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_03/BaseTaskService.cs (hunk 1 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_03/BaseTaskService.cs]: removed in hw_4

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_03/FirstTaskService.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Common.Constants;
+
+namespace BusinessLogic.Services.HW_03
+{
+    public class FirstTaskService: BaseTaskService
+    {
+        public FirstTaskService()
+        {
+        }
```
[the remark this answers is not in the document]
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_03/FirstTaskService.cs]: removed in hw_4

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_03/FourthTaskService.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading;
+using Common.Constants;
+
+namespace BusinessLogic.Services.HW_03
+{
+    public class FourthTaskService : BaseTaskService
+    {
+        public FourthTaskService()
```
[the remark this answers is not in the document]
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_03/FourthTaskService.cs]: removed in hw_4

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_03/SecondTaskService.cs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace BusinessLogic.Services.HW_03
+{
+    public class SecondTaskService : BaseTaskService
+    {
+
+        public SecondTaskService()
```
[the remark this answers is not in the document]
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_03/SecondTaskService.cs]: removed in hw_4

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_03/ThirdTaskService.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace BusinessLogic.Services.HW_03
+{
+    public class ThirdTaskService : BaseTaskService
+    {
+
+        public ThirdTaskService()
```
[the remark this answers is not in the document]
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_03/ThirdTaskService.cs]: removed in hw_4

File: ItAcademyHomeWorks/Common/Constants/HomeWorkThree.cs [hunk 7]
```diff
@@ -0,0 +1,16 @@
+﻿namespace Common.Constants
+{
+    public static class HomeWorkThree
```
[the remark this answers is not in the document]
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Constants/HomeWorkThree.cs]: thanks

File: ItAcademyHomeWorks/Common/Constants/HomeWorkThree.cs [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    {
+        public const string TextUserResultInfo = "Введите резултат";
```
[the remark this answers is not in the document]
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Constants/HomeWorkThree.cs]: fixed in hw_4

File: ItAcademyHomeWorks/HW_04_Task4/Program.cs [hunk 9]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using BusinessLogic.Services.HW_03;
+
+namespace HW_04_Task4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var hw = new FourthTaskService();
```
[the remark this answers is not in the document]
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/HW_04_Task4/Program.cs]: ok, thanks, I will fixed in hw_4

**Pull request merged**